tests/unit/test_exclude_capa.py

```diff
@@ -0,0 +1,17 @@
+"""
+Unit test for Exclude CAPA Search Filter Generator
+
+Unit test in order to make sure that the exclude dictionary with
+the ExcludeCapaSearchFilterGenerator contains `content_type` key
+"""
+import unittest
+
+from opencraft_courseware_search_filters import ExcludeCapaSearchFilterGenerator
```

Comment:
@nizarmah  You mentioned in the ticket that you mocked `lms.lib.courseware_search.lms_filter_generator.LmsSearchFilterGenerator` to get these tests to run outside the edxapp virtualenv, but I don't see that?

Did you miss a push here?

Reply:
No, not at all. So basically, the `ExcludeCapaSearchFilterGenerator` is importing the `lms.lib.courseware_search.lms_filter_generator`. Therefore, I have the `lms` folder, with the necessary path added to the `PYTHONPATH` with `tox`.

Therefore, whenever the `ExcludeCapaSearchFilterGenerator` imports the `LmsSearchFilterGenerator`, it is importing the "fake" / mock class that I created. The mock class is an actual class created instead of mocking imports with `unittest.mock` because I was having to deal with import module errors and I wasn't able to have the flexibility I wanted from the `unittest.mock`. Might have been my inexperience with it, but I was planning to partially mock the parent class without affecting the one inheriting it (the part about "not affecting the one inheriting it" is what I don't think is possible with `unittest.mock`)

Reply:
Should I mark this as resovled @pomegranited ?

Reply:
@nizarmah  Ah that's it!  There's no `test/utils/lms/lib` dir added to this PR yet :)

Reply:
@pomegranited it says they are gitignored !? oh my, I didn't pay attention to that. I'll fix it and push. Sorry about that.

Reply:
Okay, fixed it. Phew. Sorry about that.